source/_integrations/zwave_js.markdown

```diff
@@ -1093,21 +1094,21 @@ Set the log level for `zwave_js_server` to a level higher than `debug`. This can
 
 This sections lists functionality that is available in Z-Wave but that is not currently supported in Home Assistant.
 
-### Setting the controller into learn mode to receive network information
+### Setting the adapter into learn mode to receive network information
 
-In Home Assistant, it is currently not possible to set the Z-Wave controller into learn mode to receive network information from another controller.
+In Home Assistant, it is currently not possible to set the Z-Wave adapter into learn mode to receive network information from another adapter.
```

Comment:
```suggestion
In Home Assistant, it is currently not possible to set the Z-Wave adapter into learn mode to receive network information from another controller.
```
Technicality, but I'd say the 2nd "controller" was referring to the network role.